### ISSUE TITLE
PHP Version

### ISSUE THREAD

**Issue by robov** [mautic/mautic#7968]
> Mautic does not support PHP version 7.3.9 at this time. To use Mautic
Would be nice if the error mentioned the version it DOES support

**Comment by JavierLD** [comment]
Yes, me too received this message "Mautic does not support PHP version 7.3.9 at this time. To use Mautic, you will need to downgrade to an earlier version.".

The problem with this, in my case, is that I can not downgrade php version.

**Comment by RCheesley** [comment]
Please see the technical requirements page: https://www.mautic.org/download/requirements/

7.3 support is currently pending code review, the PR is here: https://github.com/mautic/mautic/pull/6991
